src/include/connection.h

```diff
@@ -127,6 +127,7 @@ struct __wt_oligarch_manager_entry {
 
     uint64_t accumulated_write_bytes;
     uint64_t checkpoint_txn_id;
+    uint64_t global_checkpoint_id;
```

Comment:
Can we rename this to known_global_checkpoint_id?

Reply:
Hmm I agree a rename makes sense but `known` doesn't quite convey what it does. What about just `read_checkpoint`?

Reply:
sgtm

Reply:
Done in bf3990e5d